src/pages/iou/ReceiptSelector/index.native.js

```diff
@@ -191,16 +191,18 @@ function ReceiptSelector({route, report, iou, transactionID, isInTabNavigator, s
                 </View>
             )}
             {cameraPermissionStatus === RESULTS.GRANTED && device != null && (
-                <NavigationAwareCamera
-                    ref={camera}
-                    device={device}
-                    style={[styles.cameraView]}
-                    zoom={device.neutralZoom}
-                    photo
-                    cameraTabIndex={pageIndex}
-                    isInTabNavigator={isInTabNavigator}
-                    selectedTab={selectedTab}
-                />
+                <View style={[styles.cameraView]}>
+                    <NavigationAwareCamera
+                        ref={camera}
+                        device={device}
+                        style={{...styles.videoContainer, display: cameraPermissionStatus !== RESULTS.GRANTED ? 'none' : 'block'}}
```

Comment:
cameraPermissionStatus check is redundant. This should be enough.
```suggestion
                        style={styles.flex1}
```
Any concern on this suggestion?